wemake_python_styleguide/violations/best_practices.py

```diff
@@ -1942,3 +1945,35 @@ class FloatKeyViolation(ASTViolation):
 
     error_template = 'Found float used as a key'
     code = 449
+
+
+@final
+class ProtectedModuleMemberViolation(ASTViolation):
+    """
+    Forbids to import protected objects from modules.
+
+    Reasoning:
+        When importing protected modules' members we break a contract
+        that authors of this module enforce.
+        This way we are not respecting encapsulation and it may break
+        our code at any moment.
+
+    Solution:
+        Do not import protected objects from modules.
+        Respect the encapsulation.
+
+    Example::
+
+        # Correct:
+        from some.public.module import FooClass
+
+        # Wrong:
+        from some.module import _protected
+        from some.module import _protected as not_protected
+
+    .. versionadded:: 0.14.0
```

Comment:
Can we please have a link to the previous violation like so:

```python
"""
Related to :class:`~ProtectedModuleViolation`.
"""
```

And the same for `ProtectedModuleViolation`, please.